what4/what4.cabal

```diff
@@ -112,19 +113,26 @@ library
     io-streams >= 1.5,
     lens >= 4.18,
     libBF >= 0.6 && < 0.7,
+    megaparsec >= 8 && < 10,
     mtl >= 2.2.1,
+    ordered-containers >= 0.2 && < 0.3,
     panic >= 0.3,
     parameterized-utils >= 2.1 && < 2.2,
+    parsec >= 3 && < 4,
     prettyprinter >= 1.7.0,
     process >= 1.2,
+    s-cargot >= 0.1 && < 0.2,
     scientific >= 0.3.6,
+    stm,
     temporary >= 1.2,
     template-haskell,
     text >= 1.2.4.0 && < 1.3,
     th-abstraction >=0.1 && <0.5,
     th-lift >= 0.8.2 && < 0.9,
     th-lift-instances >= 0.1 && < 0.2,
+    time >= 1.8 && < 1.10,
```

Comment:
Given that GHC 9.2.4 ships with `time-1.11.1.1`, it would make more sense to have the upper version bounds be `< 1.12`.

Reply:
Thanks, fixed